code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -32,6 +32,29 @@
 		else
 			to_chat(user, "<span class='notice'>[src] won't work on that.</span>")
 
+/obj/item/stack/nanopaste/afterattack(atom/M, mob/user, proximity_flag)
```

Comment:
I dont remember exactly but you said on discord that you tried with "/attack_obj()" and it didnt work?